agent/pkg/dns/dns.go

```diff
@@ -168,7 +168,7 @@ func (dns *EdgeDNS ) getFromRealDNS(req []byte, from *net.UDPAddr) {
 
 // parseNameServer gets all real nameservers from the resolv.conf
 func (dns *EdgeDNS) parseNameServer() ([]net.IP, error) {
-	file, err := os.Open("/etc/resolv.conf")
+	file, err := os.Open(hostResolv)
 	if err != nil {
 		return nil, fmt.Errorf("error opening /etc/resolv.conf: %v", err)
```

Comment:
@pengbinbin1 /etc/resolv.conf in log should modified synchronously.

Reply:
log has been modified synchronously